libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshExtra.kt

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.navigation.core.routerefresh
+
+import androidx.annotation.StringDef
+import com.mapbox.api.directions.v5.models.Incident
+import com.mapbox.api.directions.v5.models.LegAnnotation
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.base.route.RouterOrigin
+import com.mapbox.navigation.core.MapboxNavigation
+
+/**
+ * Extra data of route refresh
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+object RouteRefreshExtra {
+
+    /**
+     * Whenever route refresh controller is in idle:
```

Comment:
It's not very clear if all of these items are true or any of them. My guess is any. Could you specify this in the docs?